doc/source/whatsnew/v0.22.0.txt

```diff
@@ -60,7 +60,7 @@ Deprecations
 Removal of prior version deprecations/changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--
+- Warnings for deprecated initialization style of ``Categorical`` (``Categorical(codes, categories)``) have been removed.
```

Comment:
reference the issue (this PR is good, we also like to reference the original issue if you can find it).

``The warnings for construction of a ``Categorical`` of the form (......) have been removed``